easybuild/easyconfigs/f/FLUENT/FLUENT-18.2.eb

```diff
@@ -0,0 +1,14 @@
+name = 'FLUENT'
+version = '18.2'
+
+homepage = 'http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics/Fluid+Dynamics+Products/ANSYS+Fluent'
+description = """ANSYS FLUENT software contains the broad physical modeling capabilities needed
+to model flow, turbulence, heat transfer, and reactions for industrial applications ranging from
+air flow over an aircraft wing to combustion in a furnace, from bubble columns to oil platforms,
+from blood flow to semiconductor manufacturing, and from clean room design to wastewater treatment plants."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['FLUIDS_%(version_major)s%(version_minor)s_LINX64.tar']
```

Comment:
@shahzebsiddiqui The existing easyconfig for `FLUENT` 18.1 uses `FLUIDSTRUCTURES_%(version_major)s%(version_minor)s_LINX64.tar` instead, any idea what the difference is between `FLUIDSTRUCTURES_*.tar` and `FLUIDS_*.tar`?